indico/modules/designer/controllers.py

```diff
@@ -234,20 +242,31 @@ def _process_GET(self):
             'background_url': (bs_template.background_image.download_url
                                if bs_template and bs_template.background_image else None)
         }
+        backside_templates = (DesignerTemplate.query
+                              .filter(DesignerTemplate.backside_template_id == self.template.id)
+                              .all())
+        related_tpls_per_event = defaultdict(list)
+        related_tpls_per_categ = defaultdict(list)
+        for bs_tpl in backside_templates:
+            if bs_tpl.event_new:
+                related_tpls_per_event[bs_tpl.event_new].append(bs_tpl)
+            elif bs_tpl.category:
+                related_tpls_per_categ[bs_tpl.category].append(bs_tpl)
+        related_tpls_per_type = {
```

Comment:
You don't really use the type anywhere so why not just mix them together?

```python
for bs_tpl in backside_templates:
    related_tpls[bs_tpl.owner].append(bs_tpl)
```

If it's just for sorting to avoid mixing event/categories you could do that in the query or even split it in the jinja template using selectattr/rejectattr.

    .order_by(DesignerTemplate.event_id.isnot(None), DesignerTemplate.title)